docker/app/Dockerfile

```diff
@@ -31,6 +32,8 @@ RUN curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor |
 ENV PIP_DISABLE_PIP_VERSION_CHECK=1
 ENV PIP_ROOT_USER_ACTION=ignore
 ENV PYTHONIOENCODING UTF-8
+ENV PYTHONUNBUFFERED 1
```

Comment:
The `check output` CI step was failing, for some reason the output when running the tests was in a different order.
Setting `PYTHONUNBUFFERED=1` seems to fix that.